.reek

```diff
@@ -2,5 +2,27 @@ IrresponsibleModule:
   enabled: false
 
 exclude_paths:
-  - 'vendor'
-  - 'db/migrate'
+  - "vendor"
+  - "db/migrate"
+
+FeatureEnvy:
+  exclude:
+    - "UsersHelper#gravatar_for"
+
+"app/controllers":
```

Comment:
@hellovietnam93 
Khi chạy Reek em gặp báo lỗi sau: 
`app/controllers/users_controller.rb -- 1 warning:`
`InstanceVariableAssumption: UsersController assumes too much for instance variable '@user'`
Em tìm thấy [issue khá giống ](https://github.com/troessner/reek/issues/1133#issuecomment-256104395) và chính commiter của repo Reek nói là nếu dev rails thì nên tắt một số detector đi và [config như ở đây](https://github.com/troessner/reek/blob/master/README.md#working-with-rails)
nếu anh không đồng ý phần config này thì em sẽ bỏ và tự sửa lỗi